### PR TITLE
Adds the Energy Shotgun to HOS lockers

### PR DESCRIPTION
**Pull request by Stop-Signs**
## About the PR
Adds the energy shotgun to HOS locker, also makes the lethal bullets pass through windows

## Why / Balance
e-shotgun as far as I'm aware was nerfed AGAIN on upstream since the last time I made this PR. since the last PR for this got closed without discussion id like to open it again. The disabling barrage is now a 3 hit stun IF you hit all your shots, which you would need to be in baton range anyway to do so. The lethal shots themselves are still worse than a kammerer, however this gun can be recharged.

## Requirements

- [x] I have tested all added content and changes.
- [x] I have added media to this PR or it does not require an ingame showcase.

>

**Changelog**

:cl:
- add: Heads of Security will now have an energy shotgun in their locker.

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml
Comment (TadJohnson00) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml:1]: Heya, after some discussion with Direction they would appreciate it if you could adjust the Heat value on this file (at line 1042) from 13 to 10.

**Review by TadJohnson00 — CHANGES_REQUESTED** [review]

**Review by TadJohnson00 — APPROVED** [review]
ok, grape!

**Comment by Stop-Signs** [comment]
> ok, grape!

🍇

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml [hunk 1]
```diff
@@ -1030,15 +1030,16 @@
...
+       # - BulletImpassable
+        - Opaque # Delta-V changed from impassable so that they can go through windows
       fly-by: *flybyfixture
   - type: Ammo
   - type: Projectile
     impactEffect: BulletImpactEffectOrangeDisabler
     damage:
       types:
-        Heat: 13
+        Heat: 10
```
Comment (TadJohnson00) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml]: wait fuck comment this with delta v

almost did a fuck

**Review by TadJohnson00 — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml (hunk 1 above)
Comment (TadJohnson00) [reply Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml]: just put down "reduced damage for balance" or some shit B)

Comment (Stop-Signs) [reply Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml]: Thanks for catching that lol

**Review by TadJohnson00 — APPROVED** [review]
energy shotgun users rejoice

**Review by MilonPL — APPROVED** [review]

**Pull request merged**